test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -40,6 +42,7 @@ type KubeletManagedHostConfig struct {
 	hostNetworkPod *v1.Pod
 	pod            *v1.Pod
 	f              *framework.Framework
+	tmpfile        *os.File
```

Comment:
name this better to reflect it's the etcHost file

Reply:
done